### ISSUE TITLE
Helm reports DEPLOYED, container stuck in CreatingContainer 

### ISSUE THREAD

**Issue by ieugen** [kubernetes/helm#3468]
Hello, 

I have been working with helm and I encountered something that is confusing for me. 
I have installed an application and it showed status DEPLOYED. 
However, listing the containers with kubectl showed the container in `ContainerCreating` state. 

This was caused by the fact that I am using manually managed persistence volumes and claims ( single node cluster so I'm using hostPath persistence). 

The volume is in use by another container (another namespace) and can't be used by this container that helm creates. Thus, the status in ContainerCreated. 

I believe this is an issue with helm as it should either: 
- report CREATING instead of DEPLOYED
- report the pod statuses ?? 
 
For me it was pretty unexpected to fin out that my containers where not up and running.

**Comment by jascott1** [comment]
Hi @ieugen 

Helm templates and deploys k8s manifests as a "release" so they can be managed as one unit for upgrades and rollbacks etc. However, it does not introspect your deployment to get the final status of any entities it has deployed. From a Helm point of view, if the manifests validate and k8s accepts them then Helm's work is mostly done. I agree it would be great in some cases to do these things but the scope would be huge and in some cases perhaps its desired to deploy containers into the creating state (waiting on some condition to become true).  Hopefully this explanation helps set expectations for Helm's current role in the system. Please let us know if you have any more issues.

**Comment by ieugen** [comment]
Hi, 

Thank you for the explanation. I thing you are right. It's out of scope for Helm. 
That being said, it would be nice to have this information specified in the documentation as you worded it just above.
